spec/openapi.yaml

```diff
@@ -482,6 +572,18 @@ components:
             For example /myteam/inprotech/
             Use / to create tag in root.
           type: string
+        tagManagedBy:
+          description: >
+            Identifier of the resource that manages this tag.
+            If blank or missing, the value will be defaulted to the connector's name.
+          type: string
+        tagProperties:
```

Comment:
Suggest we follow naming convention of metadata, which looks more common name for this type of key-value map (others call it a 'tag' like AWS S3 API, which would be quite confusing in our context).

a. As we reuse same object in `AddSetTagPropertiesRequest` also, could we schema ref to a `TagMetadataSet` ?  We could then use schema ref also in schema rep of Tag itself as needed.

b. Assuming type is TagMetadataSet, could we then `tagProperties` to simply `metadata` ? 

c. Think might be clearer if we use `/tag/metadata` in place of `/tag/properties` as a tag has other properties, such as last modified timestamp and createdBy etc.

Property